ports/stm32f4/boards/stm32f411ce_blackpill/mpconfigboard.h

```diff
@@ -33,6 +33,7 @@
 #define FLASH_PAGE_SIZE             (0x4000)
 
 #define BOARD_OSC_DIV 25
+#define BOARD_NO_VBUS
```

Comment:
maybe call this NO_VBUS_SENSE to be clear

Reply:
@hierophect Could you include this change in a future PR? It doesn't have to be a standalone PR, could be part of something else.